apps/earn-protocol-landing-page/docker/Dockerfile

```
@@ -0,0 +1,48 @@
+FROM node:20-alpine
+EXPOSE 3000
+WORKDIR .
+COPY .next/standalone .next/standalone
+COPY package.json next.config.mjs .
```

Hadolint

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)